x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/use_preview_rule.ts

```diff
@@ -71,20 +71,25 @@ export const usePreviewRule = ({
     [advancedOptions]
   );
 
+  let isInvalidInterval = false;
   if (advancedOptions) {
     const timeframeDuration =
       (advancedOptions.timeframeEnd.valueOf() / 1000 -
         advancedOptions.timeframeStart.valueOf() / 1000) *
       1000;
 
     const { unit: intervalUnit, value: intervalValue } = getTimeTypeValue(advancedOptions.interval);
-    const { unit: lookbackUnit, value: lookbackValue } = getTimeTypeValue(advancedOptions.lookback);
     const duration = moment.duration(intervalValue, intervalUnit as 's' | 'm' | 'h');
```

Comment:
Is there a way to get rid of the `intervalUnit as 's' | 'm' | 'h'` type casting by properly typing `AdvancedPreviewOptions.interval`?
Same question for `lookbackUnit as 's' | 'm' | 'h'` below.

Reply:
done